content/en/blog/2023/announcing-notation-rc3.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Notation v1.0.0-RC.3 is available!
+author:  "Notation Release Team"
+date: 2023-03-08
+draft: false
+---
+
+The Notation maintainers are pleased to announce the release of Notation v1.0.0-RC.3, including Notation CLI v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release introduces a breaking change to Notation:
+
+- The default type of signature manifest is changed to image manifest. The flag `--signature-manifest` for `notation sign` command is experimental for users to store signatures using artifact manifest.
+
+Signing experience is continuously improved in this release:
+
+- Supports new flags for signing artifacts using on-demand keys
+
+### About signature manifest
+
+The signature is stored associated with signed artifacts in the OCI Distribution conformant registry. Since this release, the default type of signature manifest is changed from artifact manifest to [image manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md). The signatures of the artifacts signed by previous releases of Notation (v1.0.0-RC.1 and v1.0.0-RC.2) can still be validated using the new release.
+
+Users can still store signatures using artifact manifest by using an experimental flag `--signature-manifest`, for example:
+
+```console
+notation sign --signature-manifest artifact localhost:net-monitor@sha256:xxx
+```
+
+There are no changes on signature verification. Use `notation verify` command as usual, for example:
+
+```console
+notation verify localhost:net-monitor@sha256:xxx
+```
+
+Image manifest is commonly supported by a wide range of registries in the market. Users are strongly recommended to update to this release for using image manifest by default.
+
+### Sign artifacts using on-demand keys
+
+This release introduces a new experience of signing artifacts using on-demand keys. Here are the steps of signing:
+
+1. Create a private key and get the key identity from the Key Management System (KMS)
+2. Sign artifacts with a single command, for example:
+
+   ```console
+   notation sign --id <key_id> --plugin <KMS_plugin> localhost:5000/net-monitor@sha256:xxx
+   ```
+
+The experience is simplified for the scenario that the key is used only for one time. There is no need to add the key for later usage by `notation key add` command.
```

Comment:
I would perhaps restate this to say. This experience reduce the number of configuration steps required to setup signing. Users do not have to do the "notation key add" to start signing.

I personally think, this experience can be used to sign every time, ( not just one time) with the specified signing key.